it.js

```diff
@@ -289,10 +289,10 @@ settings.gameprivacy.header Privacy gameplay
 settings.gameprivacy.streamer Modalità streamer
 settings.gameprivacy.anon Modalità anonima
 settings.gameprivacy.hideGameTitle Nascondi titolo gioco
-settings.gameprivacy.verified Show Verified Badge
-settings.gameprivacy.premium Show Premium Badge
-settings.gameprivacy.partner Show Partner Badge
-settings.gameprivacy.rank Show Ranked Badge
+settings.gameprivacy.verified Mostra badge verificato
+settings.gameprivacy.premium Mostra premium badge
+settings.gameprivacy.partner Mostra partner badge
+settings.gameprivacy.rank Mostra ranked badge
```

Comment:
For lines 293, 294 and 295 move "badge" before the adjective like you did in line 292.
For line 295 ranked could be translated as rango.